package.json

```diff
@@ -7,6 +7,10 @@
     "@material-ui/core": "^4.11.2",
     "@material-ui/icons": "^4.11.2",
     "@reach/router": "^1.2.1",
+    "@types/jest": "^29.2.3",
+    "@types/node": "^18.11.9",
+    "@types/react": "^18.0.25",
+    "@types/react-dom": "^18.0.9",
```

Comment:
i think you'll have to update these (or `react` + `react-dom`) to match major versions. probably easier to downgrade the types deps to v17 for now